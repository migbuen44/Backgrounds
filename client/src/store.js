import { configureStore } from '@reduxjs/toolkit';
import userLoginReducer from './slices/userLoginSlice';
import loginModalReducer from './slices/loginModalSlice';
import userInfoReducer from './slices/userInfoSlice';

export default configureStore({
  reducer: {
    userLogin: userLoginReducer,
    loginModal: loginModalReducer,
    userInfo: userInfoReducer,
  },
});
