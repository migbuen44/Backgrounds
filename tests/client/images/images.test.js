import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer'
import store from '../../../client/src/store';
import Image from '../../../client/src/components/images/image';

describe('Image', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <Provider store={store}>
        <Image />
      </Provider>, div)
  })

  it('matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Image />
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  })
})