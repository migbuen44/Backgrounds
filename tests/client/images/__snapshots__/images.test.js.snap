// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image matches snapshot 1`] = `
<div
  className="lib-images-module-singleImageContainer-zbyy"
>
  <img
    alt=""
    className="lib-images-module-singleImage-3NRS click"
    onClick={[Function]}
  />
</div>
`;
